### PR TITLE
Explore teaching advisers

### PR DESCRIPTION
**Pull request by mariosh-e**
**Trello card**
https://trello.com/c/tNGVLNX6

**Context**
As we are improving the design of our subject pages, we need to update the design of the Explore teaching advisers page so it is more consistent with our other pages

**Changes proposed in this pull request**
Revise how the bursary page is organised

### CODE REVIEW

**Review by jenhadfield-dfe — DISMISSED** [review]

File: app/views/content/explore-teaching-advisers.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Explore teaching advisers
+title_paragraph: |-
+  <p>
+  Finding out which career is right for you is not always easy. How do you know what something is like if you've never done it before?</p>
```
Comment (emmaachesongray) [inline app/views/content/explore-teaching-advisers.md:5]:
```suggestion
  Our explore teaching advisers can help you discover whether the classroom is where you belong. Your dedicated adviser will give you free support and guidance, and can answer all your questions about what teaching is really like.</p>
```

Suggest removing this line, and using the first 2 sentences from the other paragraph as more effective.

File: app/views/content/explore-teaching-advisers/_article.html.erb [hunk 2]
```diff
@@ -1,56 +1,11 @@
...
-        <ul class="slashed-list">
-          <li>generation-shaping</li>
-          <li>innovative</li>
-          <li>life-changing</li>
-        </ul>
-      </span>
-    </div>
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Find your future career</h2>
+    <p>Our explore teaching advisers can help you discover whether the classroom is where you belong. Your dedicated adviser will give you free support and guidance, and can answer all your questions about what teaching is really like.</p>
```
Comment (emmaachesongray) [inline app/views/content/explore-teaching-advisers/_article.html.erb:4]: Remove first 2 sentences as they're moved above.

File: app/views/content/explore-teaching-advisers/_ending.html.erb [hunk 3]
```diff
@@ -1,44 +1,24 @@
...
-    <h2 class="heading-s">Use your subject passion</h2>
-
-    <p>
-    Bring your subject to life for your pupils. Help them be as inspired by it as you are.
-    </p>
-
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">Is teaching right for me?</h2>
+  <h3>Use your subject passion</h3>
+  <p>Bring your subject to life for your pupils. Help them be as inspired by it as you are.</p>
```
Comment (emmaachesongray) [inline app/views/content/explore-teaching-advisers/_ending.html.erb:5]: Not sure of value to users of this section, especially if main page aim is to get people to sign up to the advisers? Let's discuss tomorrow.

**Review by emmaachesongray — APPROVED** [review]
LGTM 👍

**Comment by martyn-w** [comment]
> @martyn-w @jenhadfield-dfe The **_ending.html.erb_** content is not needed anymore but I didn't want to delete the file as it says its permanent. I don't think the file is anywhere else but wanted to check the best method to do this - delete content or the whole file. I wont merge until this is confirmed. Thank you

Hi @mariosh-e , you can safely delete the file `app/views/content/explore-teaching-advisers/_ending.html.erb` but you must also remove the reference to it: i.e. delete line 17 of `app/views/content/explore-teaching-advisers.md`

**Comment by mariosh-e** [comment]
@martyn-w @jenhadfield-dfe I have deleted the ending file and deleted the reference on explore-teaching-advisers.md too. Ready for final review then I can merge. Thank you

**Review by martyn-w — APPROVED** [review]
👍 LGTM from a technical perspective - please also get a reviewer for the content

**Review by emmaachesongray — APPROVED** [review]
All good from content POV 🥇

**Pull request merged**